dgl/src/Image.cpp

```diff
@@ -92,6 +92,13 @@ void Image::loadFromMemory(const char* const rawData,
     fIsReady = false;
 }
 
+void Image::loadFromSVG(const SVG& svg) noexcept
+{
+    DISTRHO_SAFE_ASSERT_RETURN(svg.isValid(),)
+
+    loadFromMemory((const char*)svg.getRGBAData(), svg.getSize(), GL_RGBA);
```

Comment:
This is pretty bad, as the RGBA data becomes invalid when the SVG is freed. The caller needs to keep both the SVG and the Image 'alive'.

What should we do instead?

Reply:
add a parameter to allocate memory. if true, do malloc and copy data.

Reply:
It seems to me like this would be a pretty wasteful copy. I guess it could be avoided if there was an object that combined both SVG and Image.